uber/templates/jobs/staffers.html

```diff
@@ -76,4 +98,48 @@
     [L]: <i>Staffers marked with a "L" are working load-in and/or load-out</i>
 </div>
 
+
+{% if c.HAS_ACCOUNTS_ACCESS %}
+<script src="../static/js/jquery.form.js"></script>
+<script>
+var list_of_ids = [];
+
+$("#admin_settings").ajaxForm(function(message){
+    toastr.info(message);
+    setTimeout(function(){
+        location.reload(true);
+    }, 1500);
+});
+
+var checkAll = function(){
+    $.each( $("input[name^='id']"), function () {
```

Comment:
You can replace not only this but also your `uncheckAll` function with:
`$('input:checkbox:has(.checkboxclassname)').prop('checked', this.checked);`
and then convert your two buttons into a "Check All" checkbox.

Take a look at: http://jsfiddle.net/NogginBox/ScnQT/1/